feature-group-definitions/bluetooth.yml

```diff
@@ -0,0 +1,4 @@
+name: bluetooth
```

Comment:
No strong opinion from me on the ID for the feature. I thought we had a guideline against adding unnecessary prefixes like web, css, etc. though. Also, many of the web APIs start with "Web". So, we might end up with a lot of `web-*.yaml` file.
That said, people do refer to it as Web Bluetooth, not just Bluetooth.

@Elchi3 what do you think? I need another perspective on this.

Reply:
I have no strong feelings but it irked me before that when looking up things alphabetically you basically need to check B for Bluetooth and then again W for Web Bluetooth to find it. This problem becomes quite obvious when you look at https://developer.mozilla.org/en-US/docs/Web/API for example.

Also, we have `webusb`, should that be `web-usb`? 

Ultimately, I would think, that no one has thought this through from a broader perspective. So I guess we have two paths going forward:
- either web-features comes up with strong guidelines on how to name web platform capabilities (and possibly bring that back to spec authors and others), 
- or we don't want to shape existing and future naming and we use what is common today among web developers and what matches with the existing specs. This path will probably not lead to consistent names.

Reply:
> it irked me before that when looking up things alphabetically you basically need to check B for Bluetooth and then again W for Web Bluetooth to find it. This problem becomes quite obvious when you look at https://developer.mozilla.org/en-US/docs/Web/API for example.

I share your feelings here.

> or we don't want to shape existing and future naming and we use what is common today among web developers and what matches with the existing specs. This path will probably not lead to consistent names.

This sounds appealing. We've always said that web-features should match how developers think about things. On the one hand, we have an opportunity of making web-features a shared language between browser makers and web developers, by coming up with unique IDs that everyone use to talk about the web. So it would feel good to get this right. But on the other hand, I don't see how we'd land on a better naming proposal that the ones that have appeared organically and been used for years in the ecosystem already. 

So, I'm leaning towards adding the `web-` prefix.

Reply:
I agree this should have the `web-` prefix and have said all I have to say about the topic in https://github.com/web-platform-dx/web-features/issues/548#issuecomment-1999256736 :)

Reply:
@Elchi3 I filed https://github.com/web-platform-dx/web-features/issues/694 for the sorting problem, which is a problem I'm sensitive to.

Reply:
FWIW, I never imagined "name" would be the only name field, just the "conventional" (to web developers) one. I'd love to see—when relevant—`short_name` (e.g., "Bluetooth", as here) or `long_name` (e.g., "CSS Grid").